zerver/lib/github.py

```diff
@@ -61,3 +66,42 @@ def get_latest_github_release_download_link_for_platform(platform: str) -> str:
         if verify_release_download_link(link):
             return link
     return "https://github.com/zulip/zulip-desktop/releases/latest"
+
+
+def get_issue_or_pr_cache_key(owner: str, repo: str, issue_number: str) -> str:
+    return f"github_issue_or_pr_data:{owner}/{repo}/{issue_number}"
```

Comment:
This cache key should include a hash of the github token, since the contents of the response from Github also vary by that.  This means that if you change the key, you immediately start seeing responses which are limited to what that new key can access.

We need to use a hash of the github token, since the cache keys shouldn't themselves have the secret embedded in them.